packages/block-library/src/embed/embed-no-preview.native.js

```diff
@@ -168,7 +168,7 @@ const EmbedNoPreview = ( { label, icon, isSelected, onPress } ) => {
 					<Text style={ sheetTitleStyle }>
 						{ sprintf(
 							// translators: %s: embed block variant's label e.g: "Twitter".
-							__( '%s block previews are coming soon' ),
+							__( '%s embed block previews are coming soon' ),
```

Comment:
I think this message might look weird for generic embeds (example URL: https://www.amazon.com/dp/B086V2NKW6/ref=cm_sw_em_r_mt_dp_T9CZKRCGX4XFCGT31VAM) because the label's value is `Embed`, so this message will be (see attached screenshot):
`Embed embed block previews are coming soon`.

Probably we could handle that specific case to prevent duplicating the word `embed` in the message, wdyt?

<img src=https://user-images.githubusercontent.com/14905380/132740368-609231eb-cf53-4d31-aa75-7880a5a66e16.png width=350>

Reply:
Agreed! Resolved in 29a3420 dc2113f

Default Embed Info | Provider Embed Info 
--------|-------
<kbd><img src="https://user-images.githubusercontent.com/1509205/132962921-d1ce8db4-9130-41b0-99f0-bf4c8e5ff835.png" width="320"></kbd>  |      <kbd><img src="https://user-images.githubusercontent.com/1509205/132962920-4d63ca94-518e-46bc-846d-1a700e8fc126.png" width="320"></kbd>